CHANGES.md

```diff
@@ -1,5 +1,24 @@
 # Changes
 
+## edge-22.10.3
+
+This edge release adds network-validator, a new initContainer to be used when
+CNI is enabled. network-validator ensures that local iptables rules are
+working as expected. It will validate this before linkerd-proxy starts.
+network-validator replaces the noop container, runs as nobody, and drops
+all capabilities before starting.
+
+* Remove deprecation option from Gateways API (#9692)
+* Validate CNI configurations during pod startup (#9678)
+* Fix "cluster networks contains all services" fails with services with no
+  ClusterIP (#9662)
+* Fixes #9616 remove kubectl version check from `linkerd check` (#9623)
```

Comment:
This one's missing the credit attribution for the external contributor (`@ziollek`)